Library/Homebrew/cask/audit.rb

```diff
@@ -510,12 +510,13 @@ def audit_signing
           when Artifact::Pkg
             system_command("spctl", args: ["--assess", "--type", "install", path], print_stderr: false)
           when Artifact::App
-            system_command("spctl", args: ["--assess", "--type", "execute", path], print_stderr: false)
+            system_command("syspolicy_check", args: ["distribution", path], print_stderr: false)
           when Artifact::Binary
             # Shell scripts cannot be signed, so we skip them
             next if path.text_executable?
 
-            system_command("codesign",  args: ["--verify", path], print_stderr: false)
+            system_command("codesign",  args:         ["--verify", "-R=notarized", "--check-notarization", path],
```

Comment:
This change causes `chromedriver` to fail, but `1password-cli` would still pass.
My understanding is that otherwise, the only way to using the binary is to remove quarantine attributes, or install with `--no-quarantine`.